app/scripts/views/settings/emails.js

```diff
@@ -73,21 +72,11 @@ define(function (require, exports, module) {
     },
 
     _onEmailCreateError (err) {
```

Comment:
we probably don't need this function anymore. we can just put `showValidationError` into submit